docs/src/variables.md

```diff
@@ -120,6 +120,7 @@ x_interval
 julia> @variable(model, x_fixed == 4)
 x_fixed
 ```
+In the above examples, `x_free` represents an unbounded optimization variable, `x_lower` represents an optimization variable with a lower bound and so forth.
```

Comment:
Nit: Please line wrap to 80 characters (see style guide at the top of https://github.com/JuliaOpt/JuMP.jl/blob/master/docs/src/index.md).